pwa/src/apiService/resources/entity_objects.ts

```diff
@@ -0,0 +1,25 @@
+import { AxiosInstance, AxiosResponse } from "axios"
+
+export default class Entity_objects {
```

Comment:
Should be renamed to `EntityObjects`

For consistency, the file name itself should also be renamed (to `entityObjects.ts`)

Reply:
You might consider changing the name around as well (`ObjectEntity` instead of `EntityObject) because the gateway endpoint is called `/object_entities` and not the other way around.

Reply:
Also, the names within the `resource` folder are all singular, you've made it plural. If this truly is a resource that always comes in multiples it's fine, but if it's not then it should be renamed to its singular variant.